client/reader/tag-stream/controller.js

```diff
@@ -25,7 +25,9 @@ export const tagListing = ( context, next ) => {
 	const tagTitle = titlecase( trim( context.params.tag ) ).replace( /[-_]/g, ' ' );
 
 	const encodedTag = encodeURIComponent( tagSlug ).toLowerCase();
-	const streamKey = 'tag:' + tagSlug;
+
+	const streamKey =
+		context.query.sort === 'relevance' ? 'tag_popular:' + tagSlug : 'tag:' + tagSlug;
```

Comment:
If we want to load popular by default (the UI seems to indicate that in testing), we should return `tag_popular` when `query.sort` is falsy as well.

Reply:
ok, sounds good 👍